Cython/Utility/ModuleSetupCode.c

```diff
@@ -650,6 +650,16 @@ class __Pyx_FakeReference {
 #endif
 #define __Pyx_NoneAsNull(obj)  (__Pyx_Py_IsNone(obj) ? NULL : (obj))
 
+#ifndef CO_GENERATOR
+  #define CO_GENERATOR 0x20
+#endif
```

Comment:
I checked, this was already added in Python 2.2, when generators were introduced. No need to assume it could be undefined.
```suggestion
```